### ISSUE TITLE
Sign of alpha_pos and alpha_neg is steep slope loss does not match Equation 8 in the paper

### ISSUE THREAD

**Issue by Krasner** [luoyan407/predict_trustworthiness#1]
https://github.com/luoyan407/predict_trustworthiness/blob/8f394fc511b9aa31a766a30f0e1b059481aa5f76/src/oracle.py#L237

Here loss_neg contains the term `torch.exp(alpha_neg*(x[:,0]+beta_neg)/(1+torch.abs(x[:,0]+beta_neg)))` while loss_pos has the term `torch.exp(-alpha_pos*(x[:,0]-beta_pos)/(1+torch.abs(x[:,0]-beta_pos)))` however this is opposite of what's stated in equation 8 - namely `alpha_pos` should not have the negative sign which `alpha_neg` should.

![image](https://github.com/luoyan407/predict_trustworthiness/assets/7035109/09a3d73d-8b99-4952-8e15-38ad6bdb5b8e)
